ftw/simplelayout/browser/resources/integration.theme.scss

```diff
@@ -427,6 +427,15 @@ ul[class^="sl-toolbar"] {
   }
 }
 
+.SimplelayoutPortlet {
+ .sl-column {
+  min-height: 100px;
+    &:empty {
+      border: 1px dashed $color-text;
```

Comment:
Well the portlet is no in the documentEditable area. But the portlet will not be rendered in anonymous mode.